src/ThemeCatalog.js

```diff
@@ -61,7 +65,12 @@ class ThemeHero extends Component {
     this.componentInstances.forEach(ripple => ripple.destroy());
 
     // Remove the theme stylesheet if present
-    document.head.removeChild(document.getElementById('themeStylesheet'));
+    const themeStylesheetElement = document.getElementById('themeStyleSheet');
+    if (themeStylesheetElement) {
+      document.head.removeChild(themeStylesheetElement);
```

Comment:
It seems odd that we're using `document.head` here but `document.getElementsByTagName('head')[0]` above. Should we use `document.head` in both places?